app/scrapers/__init__.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import print_function
```

Comment:
sorry just want to know what is this and its purpose? why is it in every single file? not clear about it, so just curious